### PR TITLE
html-web-page - Jimmy Vu

### PR DESCRIPTION
**Pull request by Jimmy-Vu**
Screen Shot 2021-08-17 at 5 16 56 PM

### CODE REVIEW

File: html-web-page/index.html [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    customers with the highest-quality and most unique jellies on a monthly basis.
+    For the jelly lover in your life (or yourself!), this is the gift that keeps on
+    giving all year round. We offer three tiers of monethly jelly joy, and each one offers
+    their own perks. Here is a breakdown of each, as well as your monthly price:
+  </p>
+  <table>
+    <thead>
+      <th scope="col"></th>
+      <th scope="col">
+        <strong>Basic Plan</strong>
```
Comment (codingcodymiller) [inline html-web-page/index.html:35]: There shouldn't be any need for `strong` elements here, `th` is bold by default.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Great work!! One minor note for you:

File: html-web-page/index.html (hunk 1 above)
Comment (Jimmy-Vu) [reply html-web-page/index.html]: Thanks for the heads up, Cody!

**Review by codingcodymiller — APPROVED** [review]
Great work on this!! 🎉

**Pull request merged**